src/vec.cpp

```diff
@@ -899,10 +905,16 @@ vec grid_volume::dz() const {
 }
 
 grid_volume volone(double zsize, double a) {
+  if (!_wholenumber(zsize) || !_wholenumber(a))
+    master_printf_stderr(
+      "WARNING: Creating a grid_volume with non-whole numbers may result in rounding errors.\n");
```

Comment:
Maybe:
```
Warning: grid volume is not an integer number of pixels; cell size will be rounded to nearest pixel.
```

Reply:
I ran a test case with only the resolution being a floating point value and while there is still some variance in the volume levels, it is very minor. So it looks like the warning is probably not needed for that parameter, or at least not as important.

Reply:
Message changed.